Backend/app/therapy_sessions/serializers.py

```
@@ -0,0 +1,738 @@
+    """Basic therapist information for session displays"""
+    full_name = serializers.ReadOnlyField()
+    specialization = serializers.SerializerMethodField()
+    
+    class Meta:
+        model = User
+        fields = ['id', 'full_name', 'email', 'specialization']
+    
+    def get_specialization(self, obj):
+        try:
+            return obj.therapist_profile.specialization
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)